Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -209,6 +209,15 @@ - (void)configureReportbackItemCell:(LDTCampaignDetailReportbackItemCell *)repor
     reportbackItemDetailView.userDisplayNameButtonTitle = reportbackItem.user.displayName;
 }
 
+- (CGSize)reportbackItemCellSizeForIndexPath:(NSIndexPath *)indexPath {
+    UINib *reportbackItemCellNib = [UINib nibWithNibName:@"LDTCampaignDetailReportbackItemCell" bundle:nil];
+    LDTCampaignDetailReportbackItemCell *sizingCell = [[reportbackItemCellNib instantiateWithOwner:nil options:nil] firstObject];
```

Comment:
Attempted to store this as a property on the viewController instead of initializing here -- doesn't change the behavior.